default/package.json

```diff
@@ -0,0 +1,24 @@
+{
+    "name": "vanilla-typescript",
+    "version": "1.0.0",
+    "description": "JavaScript and TypeScript example starter project",
+    "main": "index.html",
+    "scripts": {
+      "start": "parcel index.html",
+      "build": "parcel build index.html"
+    },
+    "dependencies": {
+      "parcel-bundler": "1.12.5",
```

Comment:
That's the default typescript template that codesandbox proposes
It worked out of the box like that, but the libraries are outdated and trigger many npm audit warnings, so it might be worth looking into vite.